app/javascript/react/components/Navbar/Navbar.tsx

```diff
@@ -30,68 +31,91 @@ const NavItem = ({
 
 const Navbar = () => {
   const [navMenuVisible, setNavMenuVisible] = useState(false);
+  const { t, i18n } = useTranslation();
 
   return (
     <S.Header>
       <a href="https://www.habitatmap.org/" aria-label="AirCasting page">
-        <S.AircastingLogo alt="Aircasting logo" src={logo} />
+        <S.AircastingLogo alt={t("navbar.altLogo")} src={logo} />
       </a>
       <S.Container>
-        <S.BuyCTA href={urls.airbeamBuy}>get airbeam</S.BuyCTA>
+        <S.BuyCTA href={urls.airbeamBuy}>
+          {t("navbar.sections.getAirBeam")}
+        </S.BuyCTA>
         <nav>
           <S.MenuButton onClick={() => setNavMenuVisible(true)}>
-            <img src={hamburger} alt="Menu icon" aria-label="Open menu"></img>
+            <img
+              src={hamburger}
+              alt={t("navbar.altMenu")}
+              aria-label="Open menu"
+            ></img>
           </S.MenuButton>
         </nav>
       </S.Container>
       <S.NavList $isVisible={navMenuVisible}>
         <S.NavHeader>
           <a href={urls.habitatMap} aria-label="HabitatMap page">
-            <img src={habitatMapLogo} alt="Habitatmap logo" />
+            <img src={habitatMapLogo} alt={t("navbar.altHabitatLogo")} />
           </a>
           <S.Button onClick={() => setNavMenuVisible(false)}>
             <img
               src={iconNavClose}
-              aria-label="Close navigation menu"
-              alt="Close icon"
+              aria-label={t("navbar.altClose")}
+              alt={t("navbar.altClose")}
             />
           </S.Button>
         </S.NavHeader>
         <NavItem isNavTitle href={urls.airbeam}>
-          AirBeam
+          {t("navbar.sections.airBeam")}
         </NavItem>
         <S.SubNav>
-          <NavItem href={urls.airbeamUserStories}>User Stories</NavItem>
-          <NavItem href={urls.airbeamHowItWorks}>How it Works</NavItem>
-          <NavItem href={urls.airbeamFaq}>FAQ</NavItem>
-          <NavItem href={urls.airbeamUsersGuide}>User's Guide</NavItem>
-          <NavItem href={urls.airbeamBuy}>Buy it Now</NavItem>
+          <NavItem href={urls.airbeamUserStories}>
+            {t("navbar.sections.userStories")}
+          </NavItem>
+          <NavItem href={urls.airbeamHowItWorks}>
+            {t("navbar.sections.work")}
+          </NavItem>
+          <NavItem href={urls.airbeamFaq}>{t("navbar.sections.faq")}</NavItem>
```

Comment:
Maybe we can adjust, so the names are corresponding? Here I like `faq` better.

Reply:
I meant, please change the key in urls oject, so it is consistent with the translation key. :)